src/test/java/SkeltalTest.java

```diff
@@ -0,0 +1,37 @@
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import org.junit.jupiter.api.BeforeEach;
+import skeltal.Parser;
+import org.junit.jupiter.api.Test;
+import skeltal.Storage;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class SkeltalTest {
+
+    private ByteArrayOutputStream outputStreamCaptor = new ByteArrayOutputStream();
+    private PrintStream standardOut = System.out;
+
+    @BeforeEach
+    public void setUp() {
+        System.setOut(new PrintStream(outputStreamCaptor));
+    }
+
+    @Test
+    public void testParser() {
+        Parser.response("todo quick");
+        assertEquals("---------------------------------------------\n" +
```

Comment:
I believe the `+` signs should be at the start of line? Not the end. It's not explictly stated here: [https://se-education.org/guides/conventions/java/intermediate.html#layout](url), but you run into problems when implementing checkstyle later. You can configure Intellj to do this behaviour here: [https://stackoverflow.com/questions/6408462/intellij-idea-plus-sign-when-string-wrap](url)